src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+
+  if (days < 3) {
+    return days * dayCost;
+  };
+
+  if (days < 7) {
```

Comment:
There is an unnecessary semicolon after the if statement. Please remove it to keep the code consistent and clean. 😊

Reply:
changed